build.sh

```diff
@@ -124,13 +124,13 @@ install_dotnet_cli()
         __dotnetclipath=${__cli_dir}
     fi
 
-    if [ ! -f "${__cli_dir}/bin/dotnet" ]; then
+    if [ ! -f "${__cli_dir}/dotnet" ]; then
         echo "CLI could not be installed or not present."
         exit 1
     fi
 
-    echo "cat ${__cli_dir}/.version"
-    cat "${__cli_dir}/.version"
+    echo "ls ${__cli_dir}/sdk"
```

Comment:
That's covered already by line 70 in Install-cli.ps1 that didn't need to change

`say "Latest version: $RemoteVersion"`
`say "Local Version: $LocalVersion"`